api/RTCPeerConnection.json

```diff
@@ -187,17 +175,23 @@
             "samsunginternet_android": [
               {
                 "version_added": "6.0",
-                "notes": "Before Samsung Internet 8.0 the default value for the <code>RTCConfiguration.rtcpMuxPolicy</code> parameter was <code>&quot;negotiate&quot;</code>"
+                "notes": "In Samsung Internet 8.0, the default value for the <code>configuration.rtcpMuxPolicy</code> parameter changed from <code>&quot;negotiate&quot;</code> to <code>&quot;required&quot;</code>."
               },
               {
-                "prefix": "webkit",
-                "version_added": "1.5"
+                "version_added": "1.5",
+                "prefix": "webkit"
               }
             ],
-            "webview_android": {
-              "version_added": "≤37",
-              "notes": "Before WebView 63 the default value for the <code>RTCConfiguration.rtcpMuxPolicy</code> parameter was <code>&quot;negotiate&quot;</code>"
-            }
+            "webview_android": [
+              {
+                "version_added": "56",
+                "notes": "In Chrome 63, the default value for the <code>configuration.rtcpMuxPolicy</code> parameter changed from <code>&quot;negotiate&quot;</code> to <code>&quot;required&quot;</code>."
```

Comment:
Oops, WebView&rarr;Chrome here. As requested, making the note changes separately would be great.